website/static/js/meetings.js

```diff
@@ -16,20 +16,26 @@ function Meetings(data) {
              return [
                 {
                     title: 'Name',
-                    width: '50%',
+                    width: '45%',
                     sortType : 'text',
                     sort : true
                 },
                 {
                      title: 'Submissions',
-                     width : '25%',
+                     width : '15%',
                      sortType : 'number',
                      sort : true
```

Comment:
Are these inner two indented an extra space relative to the outer two on purpose?